prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -78,6 +78,15 @@ presubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20210113-cc576af-master
         command:
         - ./test/presubmit-tests-mkp.sh
+  - name: kubeflow-pipeline-mkp-snapshot-test
+    run_if_changed: "^(manifests/gcp_marketplace/.*)$"
+    cluster: build-kubeflow
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210113-cc576af-master
```

Comment:
let's use an official image or sth we maintain, I saw news k8s test images will be moved to a different place.

Reply:
I don't find much official documentation about kubekins-e2e, would you like to share a link to official image or any document? Thank you!

Reply:
I meant language specific official images like golang, python or gcp official images like cloud sdk. Pick one of them instead of kubekins e2e, because they are not official.

Reply:
Awesome, thank you Yuan for your advices! I have updated to use alpine, along with the presubmit script change which adopts this docker image: https://github.com/kubeflow/pipelines/pull/7062